src/Microsoft.DocAsCode.Common/Git/GitUtility.cs

```diff
@@ -60,7 +60,15 @@ public static GitRepoInfo GetRepoInfo(string directory)
         #region Private Methods
         private static GitDetail GetFileDetailCore(string filePath)
```

Comment:
nice catch, `GitUtility` should handle the case.

---
In reply to: [91219667](https://github.com/dotnet/docfx/pull/969#discussion_r91219667) [](ancestors = 91219667)